### ISSUE TITLE
[FEATURE/BUG] Make Smart Target functional

### ISSUE THREAD

**Issue by BlackTeaOutlands** [markdwags/Razor#45]
I have another issue with how the targetting works. You have multiple options for targetting. Im using Target next friendly & Target next non-friendly. But it doesn't really matter which one you use. THe issue is that Smart Target doesn't work in a useful way (correct me if I'm wrong). If you use Target Next Friendly, it will set the target to both Beneficial and Harmful. If you enable "Next/Previous does not set smart target", it obviously will not, but then you can't use the Next target to actually target things since nothing will ever be set. There are options to "Set Beneficial Target" and "Set Harmful Target", but those require you to manually click on bars - again, making it significantly worse/making the whole Next Target pointless.   So How Should it work? Next Friendly should only set the Beneficial Target & Next Non-Friendly should only set the harmful target. Then smart target would probably be useful.

**Comment by markdwags** [comment]
"Next/Previous does not set smart target" doesn't exist in the latest version, it was replaced with Only 'Next/Prev Friend' sets Beneficial.

When checked, if you're using Next/Prev, it will only set the harmful target. The only way beneficial target will change is if you use Next/Prev Friend.

**Comment by markdwags** [comment]
No response from user, closing issue as I believe the issue is resolved if using the latest dev build. It can be reopened if a reply comes through.

**Comment by BlackTeaOutlands** [comment]
Hey, sorry Quick I didn't see the reply here. I don't believe it seems to be working properly. It did for a few days but then it stopped. I made another bug report, not sure how to link it here.

Specifically regarding your comment up there, "Only Next/Prev Friend" sets Beneficial, it's not enough to make it functional, since for example Next Non-Friendly will set both Beneficial and Harmful. But I explained fully in the other bug report.
